src/ui/qgisapp.ui

```diff
@@ -2220,7 +2220,9 @@ Acts on all layers.</string>
      <normaloff>:/images/themes/default/mActionOffsetCurve.svg</normaloff>:/images/themes/default/mActionOffsetCurve.svg</iconset>
    </property>
    <property name="text">
-    <string>Offset Curve</string>
+    <string>Offset Curve
+  Ctrl+click with snapping to offset geometry from another layer.
+  Ctrl+2nd click to create a copy.</string>
```

Comment:
These seem to target only the "copy" scenario. Do we want to add the "simple move" one also? something like (again better suggestion welcome)
```suggestion
Click - click to offset geometry from active layer.
Ctrl+click with snapping to offset geometry from another layer.
Ctrl+2nd click to create a copy.</string>
```
or, showing all the possible actions you can do with the tool i.e.
```suggestion
Click to select geometry from active layer.
Ctrl+click with snapping to select geometry from another layer.
Ctrl+click to create a copy of the selected geometry.</string>
```

Reply:
I think having the normal procedures in the tool tip as well is a great idea. However it seems like with the possible exception of the vertex tool this isn't currently done. For example if the create polygon tool text **said left click to start right click to finish** that would save me so much time with students who are more likely to look at the tool tips then the documentation. 
Do we want to start setting this as a standard or should we reach out to the greater community?

Reply:
In my latest commit I added some doing what you suggested. Let me know what you think.

Reply:
Ping @DelazJ